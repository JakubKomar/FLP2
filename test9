A B
B C
AHOJ