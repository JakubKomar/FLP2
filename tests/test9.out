./flp22-log
Chyba - zkontrolujte Vstup
